android/java/AndroidManifest.xml

```diff
@@ -200,6 +207,14 @@
     </intent-filter>
 </service>
 
+<activity
+    android:name="com.brave.playlist.activity.PlaylistMenuOnboardingActivity"
+    android:exported="false"
+    android:launchMode="singleTask"
```

Comment:
Since this activity is singleTask without taskAffinity. It will launch in the same task.
It can be expected just want to check.
If we want to launch separate task then taskAffinity is required. To see a different instance in the recent menu.

Reply:
@sujitacharya2005 you are correct. we want to launch the activity in the same task as we would need to show onboarding only once for the playlist lifetime.